blivet/populator/populator.py

```diff
@@ -453,7 +453,7 @@ def _populate(self):
 
         self.drop_lvm_cache()
 
-        if flags.installer_mode and not flags.image_install:
+        if flags.multipath_friendly_names and not flags.image_install:
```

Comment:
Are you sure we never want to disable friendly names? The old behavior was to disable it in installer mode if requested now it would be left enabled.